Formula/spirv-tools.rb

```diff
@@ -50,8 +52,19 @@ def install
 
   test do
     cp libexec/"examples"/"main.cpp", "test.cpp"
+
+    args = []
```

Comment:
```suggestion
```
You never make use of this assignment, so there's no need to have it.

Reply:
`brew audit` fails with this error when I delete that line:

```
spirv-tools:
  * 58: col 7: Useless assignment to variable - `args`.
  * 62: col 7: Useless assignment to variable - `args`.
Error: 2 problems in 1 formula detected
```